foreman/data_refinery_foreman/surveyor/test_end_to_end.py

```diff
@@ -530,17 +530,14 @@ def test_transcriptome_redownloading(self):
             successful_processor_jobs = []
             for processor_job in processor_jobs:
                 # One of the calls to wait_for_job will fail if the
```

Comment:
We're no longer expecting one of the calls to fail, I think this comment should be updated.